plugin/pxr/cmake/macros/Public.cmake

```diff
@@ -823,7 +823,7 @@ function(pxr_monolithic_epilogue)
         PROPERTIES
             FOLDER "${folder}"
             POSITION_INDEPENDENT_CODE ON
-            PREFIX "${PXR_LIB_PREFIX}"
+            PREFIX "lib"
```

Comment:
The PREFIX for usd_m should be the same as for usd_ms - ie, CMAKE_SHARED_LIBRARY_PREFIX, which again means we could drop the setting altogether.